src/menu/create_wallet_menu.c

```diff
@@ -161,14 +173,17 @@ static void create_wallet_menu_initialize(engine_ctx_t *ctx,
 static void create_wallet_menu_handler(engine_ctx_t *ctx,
                                        ui_event_t ui_event,
                                        const void *data_ptr) {
+  bool result = 0xFF;    // default value
+  char wallet_name[NAME_SIZE] = "";
+
   if (UI_EVENT_LIST_CHOICE == ui_event.event_type) {
     switch (ui_event.list_selection) {
       case GENERATE_NEW_WALLET: {
-        create_new_wallet_flow();
+        result = create_new_wallet_flow(wallet_name);
```

Comment:
take the wallet reference as the return of the flow. let's keep the flow trigger without any parameters

Reply:
updated, re-review